### ISSUE TITLE
Example: Reachability Analysis

### ISSUE THREAD

**Issue by s8svtang** [tulip-control/dd#72]
If I understand it correctly, x_1 x_0 maps to the binary representation <x_1x_0>, e.g. x_1 = 1, x_0 = 0 => x = 2.

Then the relation (leaving out the arrows for staying on the same floor) should represent 0 <-> 1 <-> 2, or in pairs {(0,0), (0,1), (1,0), (1,1), (1,2), (2,1), (2,2)}

Instead it represents {(0,0), (0,1), (1,0), (1,1), (1,2), (2,0), (2,3)}.

For example, the reachability of 1 reaches its fix point after just one iteration.

The third line (the one for floor 2) should be (still allowing to start with the target set to floor 3, like the original relation):

"((~ x0 /\ x1) => ( (x0' /\ x1') \/ (~ x0' /\ x1') \/ (x0' /\ ~ x1') )) /\ "

or even "((~ x0 /\ x1) => ( x0' \/ x1')) /\ ", but I think it's better to explicitly name the states.

**Comment by johnyf** [comment]
Thank you for finding this error. It has been corrected on the branch `reachability_example` (I included the transitions `(2, 2)` and `(2, 1)`, as suggested). In a separate commit I changed the example to use TLA+ syntax. Using the package [`omega`](https://github.com/tulip-control/omega) can help avoid errors of this kind (an example with a reachability computation using `omega` can be found at: https://github.com/tulip-control/omega/blob/master/examples/reachability_solver.py).
